### ISSUE TITLE
[BUG] checkhealth lua functions deprecated

### ISSUE THREAD

**Issue by madskjeldgaard** [davidgranstrom/scnvim#246]
Looks like some of the health check functions will be deprecated in nvim 0.11. Here is the output of my `:checkhealth scnvim`
```
scnvim: require("scnvim.health").check()

- WARNING vim.health.report_start() is deprecated, use vim.health.start() instead. :help |deprecated|
  Feature will be removed in Nvim 0.11

scnvim ~
- WARNING vim.health.report_ok() is deprecated, use vim.health.ok() instead. :help |deprecated|
  Feature will be removed in Nvim 0.11
- OK nvim version 0.10.0
- OK sclang executable: /Applications/SuperCollider.app/Contents/MacOS/sclang
- OK scnvim classes are installed: /Users/mads/Library/Application Support/SuperCollider/Extensions/scide_scnvim
- OK keymaps are defined

scnvim documentation ~
- OK /opt/homebrew/bin/pandoc
- OK { "$1", "--from", "html", "--to", "plain", "-o", "$2" }

scnvim extensions ~
```

**Comment by davidgranstrom** [comment]
Thanks for the report!

**Comment by davidgranstrom** [comment]
Closed with #248
